lib/Sema/TypeCheckMacros.cpp

```diff
@@ -1121,18 +1119,18 @@ evaluateFreestandingMacro(FreestandingMacroExpansion *expansion,
     if (!astGenSourceFile)
       return nullptr;
 
-    BridgedString evaluatedSourceOut{nullptr, 0};
+    BridgedStringRef evaluatedSourceOut{nullptr, 0};
     assert(!externalDef.isError());
     swift_ASTGen_expandFreestandingMacro(
         &ctx.Diags, externalDef.opaqueHandle,
         static_cast<uint32_t>(externalDef.kind), discriminator->c_str(),
         getRawMacroRole(macroRole), astGenSourceFile,
         expansion->getSourceRange().Start.getOpaquePointerValue(),
         &evaluatedSourceOut);
-    if (!evaluatedSourceOut.data)
+    if (!evaluatedSourceOut.get().data())
```

Comment:
Not a huge fan of `get`. `unbridged` would match the bridging .cpp.

Reply:
Hmm yeah that seems fair enough

Reply:
Ah, I remember why I named it this way now, it's because it matches all the wrappers in SILBridging.h. If we want to change to `unbridged` we probably ought to change those too. Not sure if you have any strong opinions on this @eeckstein?

Reply:
no strong opinions. We can change it